### PR TITLE
chore(endpoints): Update region tag with product prefix

### PR DESCRIPTION
**Pull request by arbrown**

### CODE REVIEW

**Comment by arbrown** [comment]
Freezing references to snippet in cl/534553109. Please do not merge until that is submitted (or let me take care of it.) Will un-freeze references once this PR has merged.

File: endpoints/getting-started/src/IO.Swagger/app.yaml
Comment (amanda-tarafa) [inline endpoints/getting-started/src/IO.Swagger/app.yaml:4]: The region tag "configuration" is being used so you cannot just change it (that's why SnippetBot is complaining about). You first need to add the new tag on a PR, once that's merged, you need to update the docs that use the wrong tag to use the new tag. Once that CL is submitted, you need to create a new PR that deletes the old tag.

**Review by amanda-tarafa — CHANGES_REQUESTED** [review]

File: endpoints/getting-started/src/IO.Swagger/app.yaml
Comment (arbrown) [reply endpoints/getting-started/src/IO.Swagger/app.yaml:5]: Hi Amanda,

I approached this from the other angle and submitted a CL to freeze references to the snippet to a known CL and will change references once this is merged.  Having both in for now is fine though.

Thanks!

**Review by amanda-tarafa — APPROVED** [review]
SnippetBot wouldn't let me merge, it does not know about freezing the CL ;)

**Pull request merged**
